content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -2,119 +2,152 @@
+   This description is published on the public facing :doc:`website form <receiving_tickets>` where
+   customers and portal users submit tickets. The description included in this field should **not**
+   include any information that is for internal use only.
 
-Assignment & Visibility
+   .. image:: getting_started/team-description-webform.png
+      :align: center
+      :alt: View of a Helpdesk team's webform displaying the team description.
+
+Visibility & Assignment
 -----------------------
 
+*Visibility* refers to what internal users and portal users will have access to this team and its
+tickets. *Assignment* refers to how users will be assigned to handle each ticket.
+
 Determine to whom the team will be visible
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Under the :guilabel:`Visibility` section, determine who can view this team and its tickets.
 
-- :guilabel:`Invited internal users` have access to the team and tickets they are following. This
-  can be modified on each individual ticket.
-- :guilabel:`All internal users` have access to the team and all of its tickets without being a
-  follower.
-- :guilabel:`Invited portal users and all internal users` have access to the team without being a
-  follower. Portal users will only be able to access tickets that they are following.
+- :guilabel:`Invited internal users (private)` have access to the team and tickets they are
+  following. This can be modified on each individual ticket.
+- :guilabel:`All internal users (company)` have access to the team and all of its tickets without
+  being a follower.
+- :guilabel:`Invited portal users and all internal users (public)` have access to the team without
+  being a follower. Portal users will only be able to access tickets that they are following.
 
 .. example::
    A `Customer Support` team intended to handle general issues with shipping and product issues
    would have the visibility setting :guilabel:`Invited portal users and all internal users`.
    However, a `Financial Services` team handling tickets related to accounting or tax information
    would only need to be visible to :guilabel:`Invited internal users`.
 
+.. important::
+   A team's visibility can be altered after the initial configured. However, if the team changes
+   from public access to either private or company only access, portal users will be removed as
+   followers from both the team, and from individual tickets.
+
+Follow all team's tickets
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If a user should be notified about any updates regarding tickets for this team, select their name
+from the :guilabel:`Followers` drop-down. Multiple users can be selected to follow a single team.
+
+.. important::
+   External contacts can be selected in the :guilabel:`Followers` field. If the team's visibility is
+   set to :guilabel:`Invited internal users (private)`, followers will be notified about updates to
+   the team's tickets, but will not be able to view them in the portal.
```

Comment:
Is a follower not considered invited?  Why would they get a notification but not see the ticket? Just seems odd.

Reply:
These are two different issues. As per the product owner, if the team's visibility is set to private, outside contacts can still be added as Followers, meaning they will still be notified about updates to tickets. However, because the visibility trumps the follower field, they will not have access. It is not likely to be a common occurrence; however, since the system allows for it, I wanted to make note of the possibility.